utils/fo-installdeps

```diff
@@ -167,7 +167,7 @@ if [[ $RUNTIME ]]; then
             stretch|buster|sid)
                apt-get $YesOpt install postgresql-9.6 php7.0 php7.0-pgsql php7.0-cli php7.0-curl php7.0-xml php7.0-zip s-nail libboost-program-options1.62.0 libboost-regex1.62.0;;
             bionic)
-               apt-get $YesOpt install postgresql-10 php7.2 php7.2-pgsql php7.2-cli php7.2-curl php7.2-xml php7.2-zip s-nail libboost-program-options1.65.1 libboost-regex1.65.1;;
+               apt-get $YesOpt install postgresql-10 php7.2 php7.2-pgsql php7.2-cli php7.2-curl php7.2-xml php7.2-zip php7.2-mbstring s-nail libboost-program-options1.65.1 libboost-regex1.65.1;;
```

Comment:
I think this belongs in the debian control file, because php-mbstring is missing for older versions too.

Reply:
also php-xml does not seem to be covered, because it is there. for php7.2 ... but,  I have reported both at the same time?

Reply:
`php-mbstring` and `php-xml` are not part of Trusty and Jessie packages. Therefore they can not be put in the control file.